ci/do_ci.sh

```diff
@@ -246,7 +246,7 @@ elif [[ "$CI_TARGET" == "bazel.coverage" ]]; then
 
   # gcovr is a pain to run with `bazel run`, so package it up into a
   # relocatable and hermetic-ish .par file.
-  bazel build @com_github_gcovr_gcovr//:gcovr.par
+  bazel build --python_version=PY2 @com_github_gcovr_gcovr//:gcovr.par
```

Comment:
FYI, the Python version is always set by either `--host_force_python` (for host-configured targets) or the `python_version` attr of `py_binary`/`py_test` targets. If these are not explicitly specified they default to `PY3`.

So setting `--python_version` on the command line has no effect on executable Python targets. Its only use is to cause other targets (e.g. a `py_library`, or some non-Python rule) to have `PY2` or `PY3` mode for the purposes of evaluating `select()`s on the Python version and for deciding whether to use the `-py2` output directory.

Reply:
@brandjon note this is required because of the `genrule` that outputs the source file. Without this, when forcing py2 on this par_binary target, subpar believes that the `main` file is not contained in the `srcs` because it's not nested in the `-py2` output directory.

You can reproduce this by running this command with and without this flag, without it subpar fails here https://github.com/google/subpar/blob/35bb9f0092f71ea56b742a520602da9b3638a24f/subpar.bzl#L29-L30

Reply:
I see, thanks.